Texts/MIDs.h

```diff
@@ -1,5 +1,5 @@
 //MIDs.h
-//Generated by DRODUtil
+//Generated by DRODUTIL at 11/12/20 11:16PM.
```

Comment:
Revert this line's change.
Re-compile DRODUtil so it doesn't generate this timestamp anymore.  We don't need the timestamp with version control, and it creates merge conflicts for us to resolve.